pkg/controller/syncsetinstance/syncsetinstance_controller.go

```diff
@@ -71,19 +73,30 @@ type Applier interface {
 	ApplyRuntimeObject(obj runtime.Object, scheme *runtime.Scheme) (hiveresource.ApplyResult, error)
 }
 
-// Add creates a new SyncSet controller and adds it to the manager with default RBAC. The manager will set fields on the
+// Add creates a new SyncSetInstance controller and adds it to the manager with default RBAC. The manager will set fields on the
 // controller and start it when the manager starts.
 func Add(mgr manager.Manager) error {
-	return AddToManager(mgr, NewReconciler(mgr))
+	logger := log.WithField("controller", controllerName)
+	reapplyInterval := defaultReapplyInterval
+	var err error
+	if envReapplyInterval := os.Getenv(reapplyIntervalEnvKey); len(envReapplyInterval) > 0 {
+		reapplyInterval, err = time.ParseDuration(envReapplyInterval)
+		if err != nil {
+			log.WithError(err).Errorf("unable to parse %s: %s", reapplyIntervalEnvKey, envReapplyInterval)
+		}
+	}
+	log.Infof("SyncSetReapplyInterval: %s", reapplyInterval.String())
```

Comment:
Log the reapply interval as a field.
```
log.WithField("reapplyInterval", reapplyInterval).Info("Reapply interval set")
```
Note also that the "SyncSet" in the log message is redundant with the field that specifies the controller name.